content/post/057-ipfs-desktop-0-9.md

```diff
@@ -0,0 +1,65 @@
+---
+date: 2019-09-13
+url: 057-ipfs-desktop-0-9
+title: IPFS Desktop 0.9 released
+author: Henrique Dias
+---
+
+IPFS Desktop has come a long way to reach its current form. We are here to celebrate and announce the [release of 0.9.0](https://github.com/ipfs-shipyard/ipfs-desktop/releases) and share the story of this application and, of course, all the exciting new features.
+
+## Once upon a time...
+
+Back in May 2015, Juan Benet pushed the first so-called "Initial Commit" to start a project at the time named "ipfs-electron". The purpose of the app was quite simple and that hasn't changed much with time: providing a long running daemon, alongside with a GUI for managing it.
+
+![Evolution of IPFS Desktop](/057-ipfs-desktop-0-9/evolution.jpg)
+
+Regarding the application's name, it went from "ipfs-electron" to "IPFS Native Application", then to "IPFS Station", it being a much more fancier name. However, we changed it again to "IPFS Desktop" and that's where we are right now!
+
+## IPFS Desktop 0.9.0
```

Comment:
To flow more smoothly from the previous section, maybe "Where we are today: IPFS Desktop 0.9.0" or something like that?